pleskdistup/common/src/mariadb.py

```diff
@@ -56,6 +56,13 @@ def _extract_from_mysql_util(self, util_output: str):
         self.minor = int(minor_part)
         self.patch = int(patch_part)
 
+    def _extract_from_mariadb_util(self, util_output: str):
+        # String example: "mariadb from 11.6.2-MariaDB, client 15.2 for Linux (x86_64) using  EditLine wrapper"
+        major_part, minor_part, patch_part = util_output.split("from ")[1].split("-MariaDB")[0].split(".")[:3]
```

Comment:
Would it be more robust if we try to find the version with a regexp or maybe find first dot then track back the numbers...?